examples/arithmetics/src/language-server/main-browser.ts

```diff
@@ -7,6 +7,8 @@
 import { startLanguageServer, EmptyFileSystem, DocumentState } from 'langium';
 import { BrowserMessageReader, BrowserMessageWriter, createConnection, Diagnostic, NotificationType } from 'vscode-languageserver/browser';
 import { createArithmeticsServices } from './arithmetics-module';
+import { interpretEvaluations } from '../cli/evaluator';
```

Comment:
While an import from `/cli/evaluator` is not harmful by itself, it is usually assumed that a `CLI` would use `node` dependencies. Since this is the browser module, we should make clear that the code imported here does is not explicitly part of a CLI. Please move the `evaluator` to a common area that is supposed to be accessed by every consumer of the language.